src/main/java/com/linecorp/armeria/server/PrefixPathMapping.java

```diff
@@ -44,14 +44,21 @@ protected String doApply(String path) {
 
     @Override
     public int hashCode() {
-        return strVal.hashCode();
+        return prefix.hashCode();
```

Comment:
Then, the hashCodes for two `PrefixPathMapping` objects having the same values for `prefix` and the different values for `stripPrefix`, will be the same. I think it can cause a problem (the different behaviours of two objects having the same hashCodes). What do you think?